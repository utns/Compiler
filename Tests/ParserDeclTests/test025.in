type
    a = array[1..3] of byte;