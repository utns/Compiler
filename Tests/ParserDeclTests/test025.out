Symbol byte doesn't exist: line = 2 pos = 24
